src/editor/TextEditor.java

```diff
@@ -1,11 +1,115 @@
 package editor;
 
 import javax.swing.*;
+import java.awt.*;
 
 public class TextEditor extends JFrame {
+    private FileLoader fileLoader = new FileLoader();
+    static JTextArea textArea = new JTextArea();
+    static JTextField textField = new JTextField();
+
+
```

Comment:
Why too much spaces?

Reply:
Yes, I really overuse spaces, I'll pay attention to it in future, thank you!